cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperator.java

```diff
@@ -701,7 +704,7 @@ public MapAndStatus(T loadMap, K status) {
             return onNew(reconciliation, host, apiClient, kafkaRebalance, rebalanceOptionsBuilder);
         } else {
             // Stay in the current NotReady state, returning null as next state
-            return Future.succeededFuture();
+            return Future.succeededFuture(new MapAndStatus<>(null, null));
```

Comment:
@sknot-rh Hello, this line seems to resolve the NPE.

Reply:
Aha. Thank you.